sites/geohub/src/lib/helper/index.ts

```diff
@@ -22,4 +22,5 @@ export * from './updateParamsInURL'
 export * from './sleep'
 export * from './getMaxValueOfCharsInIntervals'
 export * from './distinct'
+export * from './loadMap'
```

Comment:
You can also add `getValueFromRasterTileUrlAsync.ts` to `index.ts`

Reply:
it is in the same module with getValueFromRasterTileUrl, so no need to add it